nuxeo-core/nuxeo-core-bulk/src/test/java/org/nuxeo/ecm/core/bulk/CoreBulkFeature.java

```diff
@@ -34,6 +34,7 @@
  * @since 10.2
  */
 @Deploy("org.nuxeo.runtime.kv")
+@Deploy("org.nuxeo.runtime.cluster")
```

Comment:
Is this related/needed?

Reply:
This is in a separate commit to fix a NPE